ttnn/cpp/ttnn/operations/eltwise/unary/common/unary_op_utils.cpp

```diff
@@ -247,20 +246,6 @@ std::pair<std::string, std::string> get_op_init_and_func_parameterized(
                     std::to_string((uint32_t)datatype_to_dataformat_converter((DataType)params[0])),
                     std::to_string((uint32_t)datatype_to_dataformat_converter((DataType)params[1])))};
             break;
-        case UnaryOpType::DROPOUT: {
```

Comment:
can you clarify why do you remove it from the unary op itself?

Reply:
this dropout didn't work Ill add more explanation. Overall unary op infra doesn't support any runtime or compile time args. Also it doesn't support init outside of tiles. It only creates kernels using defines.